controllers/virtualmachinesnapshot/virtualmachinesnapshot_controller.go

```diff
@@ -197,7 +197,6 @@ func (r *Reconciler) ReconcileNormal(ctx *pkgctx.VirtualMachineSnapshotContext)
 
 	if vmSnapshot.Status.Storage.Requested == nil {
 		ctx.Logger.V(4).Info("Updating snapshot's status requested capacity")
-		vmSnapshot.Status.Storage.Requested = []vmopv1.VirtualMachineSnapshotStorageStatusRequested{}
```

Comment:
Removed intentionally otherwise of `CalculateReservedForSnapshotPerStorageClass ` fails, `vmSnapshot.Status.Storage.Requested` won't be updated